### ISSUE TITLE
Blacklisted Websites preferences not being saved

### ISSUE THREAD

**Issue by paddylandau** [WorldBrain/Research-Engine#81]
I installed your Chrome extension today.

I add new websites to the Blacklisted Websites in Preferences, and press "Save", which gives the message, "Saved Preferences."

However, when I return to the Blacklisted Websites, whatever changes I have made have all disappeared.

I have tried this a few times, including after closing and restarting Chrome.

Also, [in your video tutorial](https://youtu.be/XssxGpYSH1Y), you have three Types: blank, "Regex" and "Entire Website". But, in my Preferences, there are just two: blank and "All URLs containing this text (Regex)". Is your video out of date or is my extension somehow wrong?

More information:
• Chrome version 55.0.2883.87 (64-bit)
• WorldBrain - The (Re)search-Engine version 0.0.1.8
• Linux Ubuntu 16.04 (64-bit)

Thank you.

**Comment by blackforestboi** [comment]
@paddylandau 

thanks for pointing this out!
The error is now fixed, if you press update extension in your extension overview in about 1 hour, it will work again.

We had to remove the 3 types of options, the legacy code from the project we originally forked it from had errors in these options. So for now its just the Regex.

It can be used for all the 3 options still. 

for the whole domain type in: http://guardian.co.uk
for the specific page, just the whole URL. 
for anything with a regex, just the term then.
